l20-spring/src/main/java/ua/lits/l20spring/service/UserServiceImpl.java

```diff
@@ -75,24 +97,26 @@ public void initDatabase(Resource resource) {
             return;
         }
 
-        Iterable<User> userResult = null;
+        Collection<User> userResult = null;
         try {
             List<User> userList = new ArrayList<>();
             String[] row;
+            UserDTO userDTO;
 
             for (String str : ResourceHelper.readAsList(resource)) {
-                row = str.split("\t", 5);
-                userList.add(
-                        User.builder()
-                                .firstName(row[0])
-                                .lastName(row[1])
-                                .age(Integer.valueOf(row[2]))
-                                .address(row[3])
-                                .email(row[4])
-                                .build()
-                );
-                userResult = userRepository.saveAll(userList);
+                row = str.split("\t", 6);
```

Comment:
U have a lot of magic numbers, create  constants for this, like AGE = 2, SALARY = 3, it will more readable

Reply:
Done